hack/util.sh

```diff
@@ -516,7 +516,9 @@ function install_registry {
 }
 
 function wait_for_registry {
-	wait_for_command '[[ "$(oc get endpoints docker-registry --output-version=v1 -t "{{ if .subsets }}{{ len .subsets }}{{ else }}0{{ end }}" --config=${ADMIN_KUBECONFIG} || echo "0")" != "0" ]]' $((5*TIME_MIN))
+    #if this command is hanging for you, most likely you need to either disable selinux, or leave selinux on and chcon the volume dir like in hack/test-end-to-end-docker.sh,
+    # and *possibly* ensure your TMPDIR/BASETMPDIR are not actually /tmp (but try just disabling selinux or the chcon first)
```

Comment:
it's ok for TMPDIR=/tmp if the filesystem on /tmp is not tmpfs (which fedora seems to setup by default). The chcon command actually needs to be added to core.sh. You can do it in this PR or wait for mine. We don't need to comment about it here tho.